Resources/public/paella/player/resources/style/style_dark.css

```diff
@@ -1871,3 +1871,6 @@ a:active {
   font-family: 'Istok Web', sans-serif;
   font-style: oblique;
 }
+.footPrintsContainer {
```

Comment:
`Resources/public/paella/` is only for original paella player assets.

Could you add the css rule into `/Resources/views/PaellaPlayer/index.html.twig` or `player.html.twig`.